### ISSUE TITLE
Video encoding is stuck at 98%

### ISSUE THREAD

**Issue by okulvitra** [EscolaLMS/Courses#294]
Tried to upload an .mp4 video and the process is stuck at 98%.

Video is only 10 Mo, so should not be complicated to upload, and we have enough storage on our server. 

![image](https://github.com/EscolaLMS/Courses/assets/136440102/571e26e8-8c6b-40d8-9fa3-d19c5192f05f)

To reproduce, simply try to upload a video through the video block in the course editor.

**Comment by qunabu** [comment]
@okulvitra can you give us more context for this as we cannot reproduce this

**Comment by okulvitra** [comment]
@qunabu 
We installed the infrastructure using the terraform file and everything seems to run normally. 

However, the encoding process for videos fails, for instance, I retried and it's not stuck elsewhere: 
![image](https://github.com/EscolaLMS/Courses/assets/136440102/84c3249c-cac8-4113-8b5a-2fd4e85204f7)

I wonder if the problem is coming from the server, maybe with a missing FFMEPG library or something.
The video is only 10Mo and is an .mp4 format, so should be fine to upload.
